remediation/fence-agents-remediation/fence-agents-remediation.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Fence Agents Remediation
+nav_order: 3
+has_children: true
+parent: Remediation
+---
+
+## Fence Agents Remediation
+
+Generally Available
+{: .label .label-green }
+
+[Fence Agents Remediation](https://github.com/medik8s/fence-agents-remediation#readme) (FAR) is a Kubernetes operator that uses external tools to *fence* unhealthy nodes. The tools are a set of well-known [upstream fence agents (FA)](https://github.com/ClusterLabs/fence-agents), where each one of them can be used for different environments, to fence a node using a reboot action. Moreover, FAR can be used by the [Node HealthCheck Operator](https://github.com/medik8s/node-healthcheck-operator#readme) (or any other operator that supports the external remediation API) to remediate an unhealthy node.
+
+FAR is not only fencing a node when it becomes unhealthy, it also *remediates* the node from being unhealthy to healty again. It adds a taint to evict stateless pods, fence the node with a fence agent and fully remediate it with resource deletion to remove any remaining workloads (mostly statefull workloads) after reboot.
+
+FAR is recommended when a node becomes unhealthy, and we want to completely isolate the node from a cluster, since we can’t “trust” the unhealthy node, to prevent it from accessing the shared resources like [RWO volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes). The operator is available in the Kubernetes community, [OperatorHub.io](https://operatorhub.io/operator/fence-agents-remediation), and can be installed manually as mentioned in [FAR documentation](https://github.com/medik8s/fence-agents-remediation#installation)
```

Comment:
> FAR is recommended when a node becomes unhealthy, and we want to completely isolate the node from a cluster, since we can’t “trust” the unhealthy node, to prevent it from accessing the shared resources...

While this is true in general, this isn't specific to FAR, is it? This sounds like other remediators don't do this 🤔

Reply:
>This sounds like other remediators don't do this

From the sentence beforehand I assume you mean that other operators act similar to FAR on that matter (when to use remediation).
Do you see any value of adding similar sentence to the [remediation page](https://www.medik8s.io/remediation/remediation/) if not here?

Reply:
I think we have it there already:

```
At it's core, remediation (aka. fencing) turns a question _Can our peer cause
data corruption?_ into an answer _**No!**_ by isolating it both from incoming
requests and persistent storage.
```

so it's just redundant here IMHO